lib/Attribute.js

```diff
@@ -556,7 +556,7 @@ Attribute.prototype.parseDynamo = async function(json) {
       return;
     }
     if(isSet) {
-      const set = json[type + 'S'];
+      const set = json[type + 'S'] || json;
```

Comment:
I'm concerned about this change. Isn't the Dynamoose JSON stored as something like the following?

```json
{
    "property": {
        "S": "Hello World"
    }
}
```

This change looks like it's assuming like it's the following.

```json
{
    "property": "Hello World"
}
```